libflux/flux/src/lib.rs

```diff
@@ -1123,17 +1123,31 @@ from(bucket: v.bucket)
+        let mut got = String::new();
+        for d in doc {
+            if d.name == "csv" {
+                // println!("GOT CSV DOC: {:?}", d);
+                got = format!("{:?}", d);
+                break;
+            }
+        }
         assert_eq!(stat, got);
     }
     #[test]
     //Array package's docs are accessed from their specific index in the vector of DocPackages, ensuring that they are accessible and correct
     fn ensure_docs2() {
-        let d = docs();
-        let stat = r#"PackageDoc { name: "array", headline: "Package array provides functions for building tables from flux arrays.", description: None, members: {"from": Function(FunctionDoc { name: "from", headline: "from constructs a table from an array of records.", description: "Each record in the array is converted into an output row or record. Allrecords must have the same keys and data types. Parameters is the array of records to construct a table with. Build an arbitrary tableimport \"array\"\n\nrows = [\n  {foo: \"bar\", baz: 21.2},\n  {foo: \"bar\", baz: 23.8}\n]\n\narray.from(rows: rows)\nUnion custom rows with query resultsimport \"influxdata/influxdb/v1\"\nimport \"array\"\n\ntags = v1.tagValues(\n  bucket: \"example-bucket\",\n  tag: \"host\"\n)\n\nwildcard_tag = array.from(rows: [{_value: \"*\"}])\n\nunion(tables: [tags, wildcard_tag])\n", parameters: [], flux_type: "(?bucket:string, ?bucketID:string, ?host:string, ?org:string, ?orgID:string, ?token:string) => [{A with _value:B, _time:time, _measurement:string, _field:string}]" })} }"#;
-        let got = format!("{:?}", d[0]); // the csv DocPackage location
+        let doc = docs();
+        let stat = r#"PackageDoc { name: "array", headline: "Package array provides functions for building tables from flux arrays.", description: None, members: {"from": Function(FunctionDoc { name: "from", headline: "from constructs a table from an array of records. ", description: "Each record in the array is converted into an output row or record. Allrecords must have the same keys and data types. ## Build an arbitrary table\n```\nimport \"array\"\n\nrows = [\n  {foo: \"bar\", baz: 21.2},\n  {foo: \"bar\", baz: 23.8}\n]\n\narray.from(rows: rows)\n```\n\n## Union custom rows with query results\n```\nimport \"influxdata/influxdb/v1\"\nimport \"array\"\n\ntags = v1.tagValues(\n  bucket: \"example-bucket\",\n  tag: \"host\"\n)\n\nwildcard_tag = array.from(rows: [{_value: \"*\"}])\n\nunion(tables: [tags, wildcard_tag])\n```\n\n", parameters: [ParameterDoc { name: "rows", headline: " is the array of records to construct a table with.", description: None, required: false }], flux_type: "" })} }"#;
```

Comment:
All the "/n" tags are appearing in the docs site like they are here when we are making a call to docs(), and also in the html file. Do we know what is putting them in the string representation?

Reply:
I am not too sure why that would be the case.